### PR TITLE
datadog-apm: update deprecation note to link to otlp ingest page

### PR DESCRIPTION
**Pull request by mabdinur**

### CODE REVIEW

**Comment by brettlangdon** [comment]
Is it possible to also update NPM with this change?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # dd-opentelemetry-exporter-js
 
-*Note*: This Repository has been deprecated and is no longer actively maintained.  Please refer to [Datadog Documentation](https://docs.datadoghq.com/tracing/setup_overview/open_standards/) for support options for using OpenTelemetry and Datadog.
+*Note*: This Repository has been deprecated and is no longer actively maintained.  Please refer to [OTLP Ingest in Datadog Agent](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#otlp-ingest-in-datadog-agent) for support options for using OpenTelemetry and Datadog.
 
-## OpenTelemetry Datadog Trace Exporter
+## OpenTelemetry Datadog Span Exporter
```
Comment (mabdinur) [inline README.md:5]: Change requested by Prashant

Comment (mabdinur) [inline README.md:5]: Make subtitle consistent with other documentation

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,4 +30,4 @@
 - add DatadogProbabilitySampler
 - add DatadogPropagator
 - add setup and usage instructions
-- add unit tests
\ No newline at end of file
+- add unit tests
```
Comment (brettlangdon) [inline CHANGELOG.md:33]: What is changing here?

Comment (mabdinur) [reply CHANGELOG.md:33]: I must've removed the end line. My formatter is acting up

**Review by brettlangdon — APPROVED** [review]

**Review by bengl — APPROVED** [review]

**Pull request merged**